apps/tng-radio-hub/src/Components/pages/funders/funders.tsx

```diff
@@ -0,0 +1,91 @@
+import './funders.module.css';
+
+/* eslint-disable-next-line */
+export interface FundersProps {}
+
+export function Funders(props: FundersProps) {
+    return (
```

Comment:
As in the contacts page, you need to define the data and an appropriate datastructure for funders and loop over this. You should have a separate presentational component to present the data for one such funder.